WordPress/src/main/res/layout/site_creation_form_screen.xml

```diff
@@ -15,7 +15,6 @@
         app:contentInsetLeft="@dimen/toolbar_content_offset"
         app:contentInsetStart="@dimen/toolbar_content_offset"
         app:titleTextColor="@color/white"
-        app:theme="@style/LoginTheme.Toolbar"
```

Comment:
Was this change made intentionally? I think it might have broken look of the up button on the Site Creation screen.

Reply:
![screenshot_20180612-141529](https://user-images.githubusercontent.com/2261188/41290119-34c8e226-6e4c-11e8-8ddb-f2ea4cfd676e.png)

Reply:
my bad! thanks for noticing this 👍